cli/tests/e2e/test_ci.py

```diff
@@ -1718,6 +1765,7 @@ def test_existing_supply_chain_finding(
         strict=False,
         assert_exit_code=None,
         env={"SEMGREP_APP_TOKEN": "fake_key"},
+        use_click_runner=True,  # TODO: probably because rely on some mocking
     )
     snapshot.assert_match(
```

Comment:
I take it for all the `use_click_runner=True` tests in this file, that you tried them without the click runner and they failed?

Reply:
yep